src/__tests__/RouteSelector.js

```diff
@@ -0,0 +1,122 @@
+
+import React, { createContext, useContext } from 'react';
```

Comment:
Nit: `createContext` and `useContext` are unused variables

Reply:
Resolved with 5e05629